### PR TITLE
Release v0.10.1

### PR DESCRIPTION
**Pull request by kddejong**
*Issue #, if available:*

*Description of changes:*
- Release v0.10.1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,12 @@
 - Test Ref resources to IAM Roles have good assume role documents.  Example: Lambda Function Ref refers to an IAM Role that can be assume by Lambda.
 - More Warnings around hard coded values (Regions, AccountIds) to help with the practice of reusability
 
+### 0.10.0
+###### Fixes
+- Remove dependency on regex package as it requires gcc
+- Remove rule [E3507](https://github.com/awslabs/cfn-python-lint/blob/master/docs/rules.md#E3507) because it depends on regex package
+- Support stdin for reading and testing templates
```
Comment (fatbasstard) [inline CHANGELOG.md]: Don't want to nitpick, but this is more a New feature than a fix

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,12 @@
 - Test Ref resources to IAM Roles have good assume role documents.  Example: Lambda Function Ref refers to an IAM Role that can be assume by Lambda.
 - More Warnings around hard coded values (Regions, AccountIds) to help with the practice of reusability
 
+### 0.10.0
```
Comment (fatbasstard) [inline CHANGELOG.md]: Should be `0.10.1`

**Comment by fatbasstard** [comment]
I'm missing the  version file being updated: https://github.com/awslabs/cfn-python-lint/blob/master/src/cfnlint/version.py

File: CHANGELOG.md (hunk 1 above)
Comment (fatbasstard) [reply CHANGELOG.md]: Perhaps "officially" mention Python 3.7 support?

Comment (kddejong) [reply CHANGELOG.md]: True :(  Oops. Sorry about that

**Pull request merged**